### ISSUE TITLE
"Always on" mode

### ISSUE THREAD

**Issue by lizzard77** [teroparvinen/foundry-action-pack#36]
I would prefer if the module would either remember the toggle state of the panel after a reload or in a new game, or if there was a setting to always show the panel for my players.

**Comment by teroparvinen** [comment]
I've implemented a fourth mode "Always show the tray" that is client specific (as are most of the settings). This allows each player individually to set Action Pack to stay open and automatically show when reloading. I also made a fix that automatically opens the tray if a player has the "When token selected" mode active and a token is selected when loading. This means the tray will open by default in this case because the assigned character token is selected by default.

I feel your request also contains an element of being able to promote the functionality to players by having it default to being open when a new player logs in with an assigned character. This would mean changing the way the default mode "Automatic" works with players and I'm not sure I want to make this change for all existing users. I will think this over but will mark this issue closed for now.
